examples/VisualQnA/README.md

```diff
@@ -0,0 +1,267 @@
+# VisualQnA Accuracy Evaluation
```

Comment:
shall we move these evaluation code to GenAIExamples/VisualQnA/benchmark/accuracy?